static/js/common/upload-addon.js

```diff
@@ -329,13 +329,18 @@
 
                     // Specific messages for unlisted addons.
                     var isListed = $('#id_is_listed').is(':checked'),
-                        isSideload = $('#id_is_sideload').is(':checked');
+                        isSideload = $('#id_is_sideload').is(':checked'),
+                        automaticValidation = $('#create-addon').data('automatic-validation');
                     if (!isListed) {
                       if (isSideload) {
                         $("<p>").text(gettext("Your submission will go through a manual review.")).appendTo(upload_results);
                       } else {
                         if (warnings === 0) {
-                          $("<p>").text(gettext("Your submission passed validation and will be automatically signed.")).appendTo(upload_results);
+                          if (automaticValidation === 1) {  // Automatic validation is enabled.
```

Comment:
According to the [jquery documentation on that subject](https://api.jquery.com/data/#data-html5) it does. But I didn't know it at the time I wrote the code, so thanks for noticing it ;)

Now that I know it, though, I'm going to put a boolean in there, and thus drop the `=== 1`.